### DISCUSSION TITLE
Stop straw discharge on turning

### DISCUSSION THREAD

**Discussion by JustDen1234** [Courseplay/courseplay#7383] [Ideas]
First, i love the way the new version is doing the turns while harvesting !!!

The only thing is that the discharge of strasw while turning in the corners leaves the straw not in the line of the course.
![image](https://user-images.githubusercontent.com/68871402/127473366-1d1a0119-c307-4c7b-b2fd-a39cf530596d.png)

 When picking it up using the same CP cource, the tractor will leave some straw on the field.

Is it possible to 'stop discharge straw' at turns?

**Comment by Tensuko** [comment]
If we always stop before discharge the straw in the headland turns, we would have a lot of stop and go in the turns, as there is always some stuff that we collect.
For 180° turns, we wait untill all straw is discharged.

There shouldn't be much difference, if we turn it off while in a turn. Does it rly matter if there is some straw instead of chopped straw ?

**Reply by JustDen1234** [reply]
Thanks for the reply, i understand now that this is not a simple thing to do.

And no, it does not really matter. It was just a thought :-D
